source/slang/diagnostic-defs.h

```diff
@@ -52,24 +52,27 @@ DIAGNOSTIC(    2, Error, cannotFindFile, "cannot find file '$0'.")
 DIAGNOSTIC(    2, Error, unsupportedCompilerMode, "unsupported compiler mode.")
 DIAGNOSTIC(    4, Error, cannotWriteOutputFile, "cannot write output file '$0'.")
 DIAGNOSTIC(    5, Error, failedToLoadDynamicLibrary, "failed to load dynamic library '$0'")
+DIAGNOSTIC(    6, Error, tooManyOutputPathsSpecified, "$0 output paths specified, but only $1 entry points given")
 
-DIAGNOSTIC(    6, Error, tooManyOutputPathsSpecified,
```

Comment:
This is another case where extra line breaks seem to have been inserted by `clang-format` or a similar tool. At least, I assume this was automatic...